lib/src/utils/style.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:flutter/material.dart';

```

Comment:
Can you move this file into lib/core/style? since we will be deleting the src folder

Reply:
fixed, rereview when you can